docs/modules/reference/nav.adoc

```diff
@@ -174,6 +174,7 @@
 ** xref:daemons/daemon-config-files/telemetryd.adoc[]
 ** xref:daemons/daemon-config-files/ticketer.adoc[]
 ** xref:daemons/daemon-config-files/trapd.adoc[]
+** xref:daemons/daemon-config-files/vacuumd.adoc[]
 ** xref:operation:deep-dive/admin/logging/introduction.adoc[]
```

Comment:
This shouldn't be nested within Daemons.
```suggestion
* xref:operation:deep-dive/admin/logging/introduction.adoc[]
```

Reply:
That's odd - it shouldn't be in this nav file at all. I'll remove it.

```suggestion
```

Reply:
It was added previously as a cross reference.

Reply:
Hm. Do we still need it here? The page itself is under Operation > Deep Dive > Horizon/Meridian Administration, and it confuses me a bit that we cross-reference it here rather than in another page.

@Bonrob2, what do you think?

Reply:
Ya, it does seem odd. I think we can remove it from the daemons section.